src/powershell/devcontainer-feature.json

```diff
@@ -19,9 +19,16 @@
             "type": "string",
             "default": "",
             "description": "Optional comma separated list of PowerShell modules to install."
-          }
-        },
+        }
+    },
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "ms-vscode.powershell"
```

Comment:
This change makes sense to me. However, looping in @bamurtaugh and @craiglpeters for their perspectives.

Reply:
Sounds good to me!